components/IntentionInput.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import { Modal } from 'react-native';
+import styled from 'styled-components/native'
+
+import { Entypo } from '@expo/vector-icons';
+
+const ButtonCancel = styled.TouchableOpacity`
+  padding: 10px;
+  border-radius: 50px;
+  border: red;
```

Comment:
I think you meant `border-color`

Reply:
Indeed I did. Didn't notice as no error message and the border was red.